src/shared/src/backend-supports/change-request-supports.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { ChangeRequestStatus, ImplementedChange } from '../types/change-request-types';
+
+/**
+ * This function calculates the status of a change request.
+ * @param cr An incoming change request
+ * @returns The status of the change request. Can either be Open, Accepted, Denied, or Implemented
+ */
+const calculateStatus = (
+  implementedChanges: ImplementedChange[],
```

Comment:
this function expects a lot of frontend CR fields which doesn't make sense since the point is that we're taking a backend CR and making it a frontend CR

so have this function take in the backend change request type (the same type as the input for the transformer) and then this function can just take the whole change request in